pages/doc.js

```diff
@@ -32,59 +32,69 @@ export default class Documentation extends Component {
       markdown: '',
       headings: [],
       pageNotFound: false,
-      isMenuOpen: false
+      isMenuOpen: false,
+      load: false
     }
   }
-
   componentDidMount() {
     this.loadStateFromURL()
     this.initDocsearch()
     window.addEventListener('popstate', this.loadStateFromURL)
-    this.ps = new PerfectScrollbar('#sidebar-menu', {
-      // wheelPropagation: window.innerWidth <= 572
-      wheelPropagation: true
-    })
-  }
-
-  componentDidUpdate() {
-    this.ps.update()
   }
-
   componentWillUnmount() {
     window.removeEventListener('popstate', this.loadStateFromURL)
   }
-
-  loadStateFromURL = () => {
-    const { pathname } = window.location
-    const sectionURL = pathname.split('/')[2] // match section from URL
-    const sectionIndex = sidebar.findIndex(
-      section => (section.slug || kebabCase(section.name)) === sectionURL
-    )
-    if (sectionIndex === -1) {
-      sectionURL
-        ? this.setState({ pageNotFound: true })
-        : this.onSectionSelect(0)
+  toString(method, str) {
+    switch (method) {
```

Comment:
why so complicated? is there a second method somewhere?

Reply:
This is in case we need to convert the string to another (filetourl, urltofile, nametofile, filetoname), etc. Since everything is working now, we have removed this function.

Reply:
forgot to commit?

Reply:
yes, forgot